packages/11ty/_plugins/transforms/outputs/epub/manifest.js

```diff
@@ -61,7 +61,11 @@ export default (eleventyConfig) => {
       return
     }
 
-    return path.join(imageDir.replace(/^\//, ''), image)
+    // Remove leading absolute pathing
+    let imageDirPath = path.normalize(imageDir)
+    if (imageDirPath.split(path.sep).at(0) === '') imageDirPath = imageDirPath.split(path.sep).slice(1).join(path.sep)
```

Comment:
💭 this might be more clear?
```suggestion
     if (imageDirPath.startsWith('/')) {
       imageDirPath = imageDirPath.slice(1)
     }
```

Reply:
@mphstudios The bracing and removal of complexity is fine / welcome but the check needs to be platform-specific ala:
```
if (imageDirPath.startsWith(path.sep)) { imageDirPath = imageDirPath.slice(1) }
```